### PR TITLE
Final part

### PR DESCRIPTION
**Pull request by sergio-usma**
- The "About myself" section (placed after the projects and before the contact form) with content aligned using Flexbox
- CSS Grid and Flexbox have been used to lay out the elements in the "Projects" section.
- CSS Grid has been used in the main container containing all the cards, and Flexbox has been used to lay out an individual card.
- The "Projects" and "About myself" sections use ul and li for categories. HTML, CSS, and Javascript are list items for the "Projects" section, and Languages, Frameworks, and Skills are list items for the "About myself" section.
- The detail modal window that requires JavaScript has not been implemented.
- Using the templates in Figma, the design has been followed as closely as possible (e.g., font, colors, images, text, margins).
- Button interactions (hover, etc.) have been implemented.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,156 +73,270 @@ <h1>
...
+        <!-- 2.3. Project #3 -->
 
-      <!-- 2.2. Project #2 -->
+        <!-- 2.3.1 Preview image -->
+        <div class="preview-img">
+          <img src="./media/Project-2.png" width="320px" />
+        </div>
+        <!-- 2.3.2 Description -->
+        <div class="pr-description">
+          <h2>Project name goes here</h2>
```
Comment (tufoinnkuo10) [inline index.html:135]: - [ ] Great job.. please, there is this extra margin right on your projects and headline section making your mobile version to have a scroll from left to right. Please fix that.

demo4

File: index.html [hunk 2]
```diff
@@ -72,156 +73,270 @@ <h1>
...
-      <!-- End of project -->
+    </section>
+
+    <section id="second-section" class="about-section">
+      <div class="about-info">
+        <h2>About</h2>
 
-      <!-- 2.4. Project #2 -->
+        <p>
+          I can help you build a product , feature or website Look through some
```
Comment (tufoinnkuo10) [inline index.html:234]: - [ ] please, kindly centralize the about section. Both the left and right margins should be the same. For now, the right is wider than the left. Please fix that.  
Screenshot (923)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @sergio-usma 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (sergio-usma) [reply index.html:142]: Ok! Done!

File: index.html (hunk 2 above)
Comment (sergio-usma) [reply index.html:241]: Ok! Thank you very much for your advice. You help me to be better!

**Review by amejid — APPROVED** [review]
Hi @sergio-usma, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**